server/controllers/Menu.js

```diff
@@ -9,6 +9,44 @@ class Menu {
     const menus = rows;
     res.status(200).json({ state: 'Succesful', message: 'Retrieved all menus', menus });
   }
+
+  static async postMenu(req, res) {
+    // add if verified jwt is not admin return unauthorized 401
+    console.log(req.aDecodedUser);
+    const hasAdminPermission = req.aDecodedUser;
+
+    if (hasAdminPermission.role !== 'admin') {
```

Comment:
Don't you think this should be done even before the controller is called? Make use of middlewares to achieve this.